src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -5,8 +5,33 @@
     public int value;

```

Comment:
Immutable - should be final (setter unnecessary)
After encapsulation fields should be private